tests/unit/records/test_permissions.py

```diff
@@ -85,3 +87,52 @@ def test_file_permissions(app, db, record_with_files_creation,
 
         res = client.get(file_url)
         assert res.status_code == expected
+
+
+def test_rat_token(app, db, rat_generate_token, closed_access_record):
+    """Test access via RAT."""
+    rat_generate_token = rat_generate_token['token']
+    record = closed_access_record
+    record['owners'] = [rat_generate_token.user_id]
+    record['_deposit']['owners'] = [rat_generate_token.user_id]
+    record.commit()
+    db.session.commit()
+
+    rat_token = jwt.encode(
+        payload={
+            'iat': datetime.utcnow(),
+            'sub': {'deposit_id': record['_deposit']['id']},
+        },
+        key=rat_generate_token.access_token,
+        algorithm='HS256',
+        headers={'kid': str(rat_generate_token.id)},
+    )
+
+    with app.test_client() as client:
+        file_url = url_for(
```

Comment:
It would be nice to also add a second record under this user and try to access it with the token to avoid future misbehavior.